src/document/operation/changeoperation.js

```diff
@@ -130,6 +134,12 @@ CKEDITOR.define( [
 					value.node.setAttr( newAttr );
 				}
 			}
+
+			return {
+				range: this.range,
+				oldAttr: oldAttr ? oldAttr : undefined,
```

Comment:
Agreed. This is a leftover after the previous idea of having separate events for add, removing and changing attribute.